translations/fr/try_ruby_110.md

```diff
@@ -0,0 +1,24 @@
+---
+lang:   EN
+title:  Les garçons sont différents des filles
+answer: \[\]
+ok:     Bon, d'accord, voilà une liste vide.
+error:
+---
+
+Et les nombres sont différents des strings.
+
+Bien qu'il soit possible d'utiliser des methodes sur n'importe quel
+objet avec Ruby, certaines methodes ne fonctionnent qu'avec certains
+types de choses. Mais il est toujours possible de passer d'un type à un
+autre avec les methodes "to" to Ruby.
+
+- __to_s__ convertit les choses en __s__trings
+- __to_i__ convertit les choses en __i__ntegers (nombres)
+- __to_a__ convertit les choses en __a__rrays
+
+__Qu'est-ce qu'un array?!__
+
+Ce sont des listes. Saisi une paire de crochets:
```

Comment:
I would have answered the singular question by a singular answer: `C'est une liste.`